### PR TITLE
Docs(ci): Introduce manual testing section with Netlify links tweak

### PR DESCRIPTION
**Pull request by literat**
## Description

  * @see: https://gist.github.com/adamkudrna/694f3048c1338f07375b9b8af24afe2f
  * Many thanks to @adamkudrna

### Issue reference

https://jira.lmc.cz/browse/DS-611

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md).
- [ ] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Review by crishpeen — COMMENTED** [review]
I am not really sure that this should be part of our repository 🤷

**Comment by literat** [comment]
> I am not really sure that this should be part of our repository 🤷

Why not? I think it is cool to have here some tweaks that can help us with QA.

**Comment by literat** [comment]
As we agreed, only the link to the Gist was added. Plus some fixes with Grammarly.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,9 +101,9 @@ The following is the list of supported scopes:
   - `web-react`
   - `web-twig`
 - Repository-wide:
-  - `ci`: used for changes that affects Continuous Integration process and builds
-  - `repo`: used for repository wide changes
-  - none/empty string: useful for `Test` and `Refactor` changes that are done across all packages (e.g. `Test: Add missing unit tests`) and for docs changes that are not related to a specific package (e.g. `Docs: Fix typo in tutorial`).
+  - `ci`: used for changes that affect the Continuous Integration process and builds
+  - `repo`: used for repository-wide changes
+  - none/empty string: useful for `Test` and `Refactor` changes that are done across all packages (e.g. `Test: Add missing unit tests`) and for docs changes that are not related to a specific package (e.g. `Docs: Fix `typo in `the `tutorial`).
```
Comment (crishpeen) [inline CONTRIBUTING.md:106]: backtick madness. Please fix.

**Review by crishpeen — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (literat) [reply CONTRIBUTING.md:106]: Hmm, Grammarly extension fixes. Always broke something. :-/

Comment (literat) [reply CONTRIBUTING.md]: Thanks for noticing.

**Pull request merged**